src/sql/workbench/services/query/common/queryManagement.ts

```diff
@@ -266,6 +268,15 @@ export class QueryManagementService implements IQueryManagementService {
 		});
 	}
 
+	public notifyConnectionUriChanged(newUri: string, oldUri: string): Promise<void> {
```

Comment:
This should also probably be `changeConnectionUri` since it's actually updating its own URI mappings - it's not just notifying that it was changed.